### ISSUE TITLE
Avoid assuming by default maxAge = 7 days

### ISSUE THREAD

**Issue by aletorrado** [renefloor/flutter_cache_manager#86]
Hi! I've noticed that data is being proactively cached when no Cache-Control headers is present, and there's no way to disable or modify this behavior, even when the `maxAgeCacheObject` parameter is lower than this.

I think that `maxAgeCacheObject` should be honored in this case.

EDIT: I've done some debugging, and it seems to me like items are being removed from cache correctly but kept on some memory cache that is discarded when the application is terminated.

**Comment by renefloor** [comment]
The image widget kept a cache separated from the cache manager. I fixed that in 1.1.0, now they use the same memory cache and when images are cleared from the cache they should be reloaded. If not you can reopen this issue.

**Comment by olafurorng** [comment]
I'm trying to set the `maxAgeCacheObject` to 2 minutes, but the object is still being cached (when I'm using the `flutter_cached_network_image` package).

(note: when I use `Image.network`, the image is not cached)

Is there something which I'm missing or what could be the reason the image is being cached for longer than 2 minutes?

My custom cache manager:
```
class ImageCacheManager extends BaseCacheManager {
  static const key = "libCachedImageData";

  static ImageCacheManager _instance;

  factory ImageCacheManager() {
    if (_instance == null) {
      _instance = new ImageCacheManager._();
    }
    return _instance;
  }

  ImageCacheManager._() : super(key, maxAgeCacheObject: Duration(minutes: 2));

  Future<String> getFilePath() async {
    var directory = await getTemporaryDirectory();
    return p.join(directory.path, key);
  }
}
```
